ste/mgr-JobMgr.go

```diff
@@ -97,14 +97,19 @@ type IJobMgr interface {
 	AddSuccessfulBytesInActiveFiles(n int64)
 	SuccessfulBytesInActiveFiles() uint64
 	CancelPauseJobOrder(desiredJobStatus common.JobStatus) common.CancelPauseResumeResponse
+
+	// Cleanup Functions
+	JobStatusMgrClean()
+	JobPartsMgrsDelete()
```

Comment:
From a usability point of view, perhaps it's better to just have one cleanup method that does everything, so that the caller doesn't have to worry about calling the different methods in the right order and whatnot

Reply:
@zezha-msft We have already one function in jobsAdmin file which call the these functions (JobMgrCleanup())